plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -448,4 +448,25 @@ public void testQuotesInColumnName()
         assertEquals(extractPql(dynamicTable, TupleDomain.all()), expectedPql);
         assertEquals(dynamicTable.getTableName(), tableName);
     }
+
+    @Test
+    public void testQueryOptions()
+    {
+        String tableName = realtimeOnlyTable.getTableName();
+        String tableNameWithSuffix = tableName + REALTIME_SUFFIX;
+        String query = """
+                SELECT FlightNum
+                FROM %s
+                LIMIT 50;
+                SET skipUpsert=true;""".formatted(tableNameWithSuffix);
+        DynamicTable dynamicTable = buildFromPql(pinotMetadata, new SchemaTableName("default", query), mockClusterInfoFetcher, TESTING_TYPE_CONVERTER);
+        String expectedPql = """
+                SELECT "FlightNum" \
+                FROM %s \
+                LIMIT 50;
+                SET skipUpsert = true;
```

Comment:
Does Pinot use this option in the above SELECT statement even in this order (SELECT → SET)?

Reply:
Order doesn't matter in a multi statement query

Reply:
updated the test to move SET statement front